py-wacz/tests/test_wacz.py

```diff
@@ -27,44 +16,32 @@ def setUpClass(self, mock_now):
         with zipfile.ZipFile(os.path.join(TEST_DIR, 'example.wacz'), "r") as zip_ref:
             zip_ref.extractall("tests/fixtures/unzipped_wacz")
             zip_ref.close()
-        
+
         self.wacz_file = os.path.join(TEST_DIR, 'example.wacz')
         self.warc_file = os.path.join(TEST_DIR, 'example-collection.warc')
-        self.wacz_archive = os.path.join(os.path.dirname(os.path.realpath(__file__)), "fixtures/unzipped_wacz/archive/example-collection.warc")
-        self.wacz_index_cdx = os.path.join(os.path.dirname(os.path.realpath(__file__)), "fixtures/unzipped_wacz/indexes/index.cdx.gz")
-        self.wacz_index_idx = os.path.join(os.path.dirname(os.path.realpath(__file__)), "fixtures/unzipped_wacz/indexes/index.idx")
-        self.wacz_yaml = os.path.join(os.path.dirname(os.path.realpath(__file__)), "fixtures/unzipped_wacz/webarchive.yaml")
+        self.wacz_archive = os.path.join(os.path.dirname(os.path.realpath(__file__)), "fixtures/unzipped_wacz/data/archive/example-collection.warc")
```

Comment:
Frictionless requires all data to be in `data` subdirectory, right?

Reply:
yup!